tasks/build/package.mjs

```diff
@@ -47,8 +47,8 @@ export default async function build () {
     destPath: paths.package
   })
 
-  // Copy GOV.UK Frontend JavaScript (ES modules)
-  await files.copy('**/!(*.test).mjs', {
+  // Compile GOV.UK Frontend JavaScript (ES modules)
+  await scripts.compile('!(*.test).mjs', {
```

Comment:
Notice the glob pattern has the suffix `**/` removed

Rollup will automatically find `import` modules for us